optimize/pom.xml

```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
is this right?

Reply:
https://github.com/camunda/camunda/pull/21634